data/sql_incremental_aggregates/update_schedule_a_fulltext.sql

```diff
@@ -4,15 +4,18 @@ begin
     where sched_a_sk = any(select sched_a_sk from ofec_sched_a_queue_old)
     ;
     insert into ofec_sched_a(
-        select
-            *,
-            to_tsvector(contbr_nm) as contributor_name_text,
-            to_tsvector(contbr_employer) as contributor_employer_text,
-            to_tsvector(contbr_occupation) as contributor_occupation_text,
-            is_individual(contb_receipt_amt, receipt_tp, line_num, memo_cd, memo_text)
+        select distinct on (sched_a_sk)
+            new.*,
+            to_tsvector(new.contbr_nm) as contributor_name_text,
+            to_tsvector(new.contbr_employer) as contributor_employer_text,
+            to_tsvector(new.contbr_occupation) as contributor_occupation_text,
+            is_individual(new.contb_receipt_amt, new.receipt_tp, new.line_num, new.memo_cd, new.memo_text)
                 as is_individual,
-            clean_repeated(contbr_id, cmte_id) as clean_contbr_id
-        from ofec_sched_a_queue_new
+            clean_repeated(new.contbr_id, new.cmte_id) as clean_contbr_id
+        from ofec_sched_a_queue_new new
+        left join ofec_sched_a_queue_old old on new.sched_a_sk = old.sched_a_sk and old.timestamp > new.timestamp
+        where old.sched_a_sk is null
```

Comment:
Yeah! The goal here is to add all records in the added queue that aren't also in the deleted queue with a later timestamp. For example, if somebody at FEC adds a record and then deletes it a minute later, it's going to be in both queues, but we don't want to add it to our ofec_* table. We're using a left join because we want to include all records in the added queue, even if there's no corresponding record in the deleted queue. And we're joining on primary key and relative timestamp to get deleted records with the same key that were deleted *after* the record was added to the added queue. Then we're filtering on `old.sched_a_sk is null` so that we only get records in the added queue that weren't subsequently deleted.

Hope that makes sense. If you can think of a simpler way to implement the behavior, I'm happy to revise. Otherwise, I can add comments here to explain the logic to our future selves.